jetty-util/src/main/java/org/eclipse/jetty/util/IncludeExcludeSet.java

```diff
@@ -180,12 +180,10 @@ public boolean test(P t)
      */
     public Boolean isIncludedAndNotExcluded(P item)
     {
-        if (_excludePredicate.test(item))
+        if (_excludePredicate.test(item)) {
             return Boolean.FALSE;
-        if (_includePredicate.test(item))
-            return Boolean.TRUE;
-
-        return null;
+        }
+        return _includes.size() == 0 || _includePredicate.test(item);
```

Comment:
This is definitely a wrong change!  The point of this method is that it is a try state - sometimes we depend on the null return